ansible_collections/arista/avd/tests/unit/filters/test_snmp_hash.py

```diff
@@ -2,19 +2,14 @@
 
 __metaclass__ = type
 
-from contextlib import contextmanager
-
 import pytest
 from ansible.errors import AnsibleFilterError
 
 from ansible_collections.arista.avd.plugins.filter.snmp_hash import PRIV_KEY_LENGTH, FilterModule, get_hash_object, key_from_passphrase, localize_passphrase
 
-f = FilterModule()
-
+from ..conftest import does_not_raise
```

Comment:
I don't like this, since it is not part of a package. It is just standalone scripts.
All our tests are written slightly different, so I don't think we should overdo it by sharing 5 lines in a common lib. But if you wish, then at least use absolute import.

Reply:
Addressed in 0dbeecf - now using contextlib nullcontext and remove this file all together

Also moved the new tests to the correct place as per ansible recommendation